community-operators/sysdig/sysdig-operator.v1.4.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,92 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: sysdig-operator.v1.4.0
+  namespace: placeholder
+  annotations:
+    capabilities: "Basic Install"
+    categories: "Security, Monitoring"
+    certified: "false"
+    description: |-
+      [Sysdig](https://www.sysdig.com/) is a unified platform for container and microservices monitoring, troubleshooting, security and forensics. Sysdig platform has been built on top of [Sysdig tool](https://sysdig.com/opensource/sysdig/) and [Sysdig Inspect](https://sysdig.com/blog/sysdig-inspect/) open-source technologies.
+    containerImage: docker.io/sysdiglabs/sysdig-operator:1.4.0
+    createdAt: 2019-03-08T18:30:00Z
+    support: Sysdig, Inc.
+spec:
```

Comment:
This operator needs a `spec.installModes` field to work with OLM, as described here: https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Documentation/design/building-your-csv.md#operator-metadata
More info about OperatorGroups can be found here: https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Documentation/design/operatorgroups.md